internal/graph/graph.go

```diff
@@ -19,6 +21,29 @@ var (
 	ErrResolutionDepthExceeded = errors.New("resolution depth exceeded")
 	ErrTargetError             = errors.New("graph: target incorrectly specified")
 	ErrNotImplemented          = errors.New("graph: intersection and exclusion are not yet implemented")
+
+	RelationshipIngressTransformer = cmp.Transformer("Sort", func(in []*RelationshipIngress) []*RelationshipIngress {
```

Comment:
Is this only used in the graph_test.go?  If so would it make sense to define it there?

Reply:
I don't have a preference. I mimicked what we did for `TupleKeyCmpTransformer` which is defined in `testutils` (but i can't put it next to it or i'll create a cycle of imports)